Flank/src/main/java/com/walmart/otto/Flank.java

```diff
@@ -71,6 +85,17 @@ public void start(String[] args)
     printExecutionTimes(startTime);
   }
 
+  private void aggregateTestReports(File dir) {
+    try {
+      new ReportsAggregator(configurator, gsutilTool).aggregate(dir.toPath());
+    } catch (XmlReportGenerationException
+        | HtmlReportGenerationException
+        | IOException
+        | InterruptedException e) {
+      e.printStackTrace(); // don't fail the build for the reports
```

Comment:
if we don't fail to aggregate the reports, will the reports still exist in non-aggregated form? otherwise it would be very hard to debug a build.

Reply:
The source reports are not deleted/changed. 

The rationale for not failing the build in case of an exception during the report processing is that the processing itself might be buggy at this point and we don't want to fail a potentially successful test run. We could be more strict about this in the future. WDYT?

Reply:
sounds good to me! although this is probably a better question for the maintainers.